src/client/comments/commentsActions.js

```diff
@@ -97,7 +97,9 @@ export const sendComment = (parentPost, body, isUpdating = false, originalCommen
   const permlink = isUpdating
     ? originalComment.permlink
     : createCommentPermlink(parentAuthor, parentPermlink);
-  const jsonMetadata = { tags: [category], community: 'busy', app: `busy/${version}` };
+  const jsonMetadata = isUpdating
+    ? JSON.parse(originalComment.json_metadata)
```

Comment:
We should handle the case where json_metadata is not a real json. JSON.parse will fail if this happen with your implementation.

Reply:
You think https://github.com/busyorg/busy/blob/a2d948db624af33eb01c1237bfb3779df6539e0c/src/client/helpers/formatter.js is the right place to put a `isJson` helper function or something?

Reply:
I don't want to put it right in `commentsAction.js`

Reply:
We might want to extract method tryParseJSON that uses lodash's `try` and `isError` to return parsed JSON or empty object.